core/extraction/towns_fund_round_three.py

```diff
@@ -885,6 +885,7 @@ def extract_outcomes(df_input: pd.DataFrame, project_lookup: dict, programme_id:
     outcomes_df = pd.concat([outcomes_df, pd.DataFrame(df_input.values[27:37], columns=header_row_combined)])
     # If indicator name or project ref is empty - drop row (cannot map to DB table)
     outcomes_df = drop_empty_rows(outcomes_df, ["Indicator Name"])
+    outcomes_df = drop_empty_rows(outcomes_df, ["Relevant project(s)"])
```

Comment:
From my understanding this only needs to be executed for Round 3 right? Can you add a param and preserve the Round 4 functionality as part of this PR